Makefile

```diff
@@ -143,8 +143,8 @@ else
 endif
 	@echo Building k8sapiserver...
 	mkdir -p bin
-	$(DOCKER_RUN) -e CGO_ENABLED=$(CGO_ENABLED) $(CALICO_BUILD) \
-		sh -c '$(GIT_CONFIG_SSH) go build -v -i -o $@ -v $(LDFLAGS) $(PACKAGE_NAME)/cmd/apiserver'
+	$(DOCKER_RUN) -e CGO_ENABLED=$(CGO_ENABLED) $(CALICO_BUILD)-$(ARCH) \
```

Comment:
Other repos don't use `$(CALICO_BUILD)-$(ARCH)` and instead just use `$(CALICO_BUILD)` - what's the reason for doing so here?

Reply:
This is the binary build process and CGO_ENABLED is true.
`$(ARCH)` runs the build process inside an arm64 container.

We could omit that and use `CC` flag to cross compile the binary using `gcc` cross compiler.